gokart/gcs_obj_metadata_client.py

```diff
@@ -71,30 +81,63 @@ def add_task_state_labels(
             if update_response is None:
                 logger.error(f'failed to patch object {obj} in bucket {bucket} and object {obj}.')
 
+    @staticmethod
+    def _normalize_labels(labels: Optional[dict[str, str]]) -> dict[str, str]:
```

Comment:
this annotation seems to be invalid, since `_normalize_labels` tries to convert `Optional[dict[str, Any]` to `Optional[dict[str, str]`
```suggestion
    def _normalize_labels(labels: Optional[dict[str, Any]]) -> dict[str, str]:
```